ITPC21 - DAA/lcs_tabulation.cpp

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+using namespace std;
+static int arr[7][8];
+void lcs(string X, string Y, int i, int j)
```

Comment:
use string&, otherwise it copies the entire X and Y
and complexity becomes O(n * m * (n + m))

```suggestion
void lcs(string &X, string &Y, int i, int j)
```